Dockerfile-worker

```diff
@@ -1,8 +1,18 @@
 FROM python:3.10-slim
 
-# install gfortran via apt and clean cache
+# install libraries required to run simulators via apt and clean cache
 # skipcq: DOK-DL3008
-RUN apt-get update && apt-get install -y --no-install-recommends gfortran && rm -rf /var/lib/apt/lists/*
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    libexpat1-dev \
```

Comment:
Can we make installation of TOPAS optional. It can be driven by presence of necessary env variables. If the variables are set then the necessary apt packages are being installed. This could either be a smart one-liner or the installation of TOPAS requirements can be moved to a bash script.

This way we will have smaller container image and faster deploy

Reply:
I made it so if any of TOPAS variables are not set then the libraries are not downloaded

Reply:
good